ecnet2/ecnetd.lua

```diff
@@ -1,10 +1,8 @@
-local redrun = require "redrun"
 local constants = require "ecnet2.constants"
 
 --- The global daemon state.
---- @class ecnet2.EcnetdState
---- @field handlers table<string, fun(etc: string, side: string)>
-local state
+
+local handlers = {}
```

Comment:
```suggestion
local handlers = setmetatable({}, { __mode = "v" })
```
Handler mappings must be dropped whenever their handler function gets dropped, otherwise you create a memory leak when dropping existing listeners or connections.

Reply:
ah good catch I must have missed that in moving stuff around